lib/index.js

```diff
@@ -169,13 +171,13 @@ class ProbePlugin {
       event = request.input.body.event;
     debug(`Received measure for monitor ${event}`);
 
-    this.eventMapping.monitor[event].forEach(probe => {
+    for (const probe of this.eventMapping.monitor[event]) {
```

Comment:
Why use iterator over forEach ?

Reply:
to be consistent with the code of kuzzle core.

* it is easier to read
* it is faster